File: PROGRAMAS/4_CALCULO_DE_PI/calculo_de_pi.r

a = 0 # intervalo
b = 1 # intervalo
n = 100000 # tamaño de la partición.
h = (b-a) /n # fineza de la partición.
suma = 0 # aproximación de la integral

for(i in 1:n-1) suma = suma + sqrt(1 - (a+i*h) **2)

suma = h*(0.5 + suma)
sprintf(4*suma, fmt = "%#.15f")

File: PROGRAMAS/4_CALCULO_DE_PI/calculo_de_pi.py
import numpy as np

a = 0
b = 1
n = 100000
h = (b-a) / n
suma = 0

for i in range(n):
    suma += np.sqrt(1- (a+i*h) ** 2)

suma = h * ((1-0)/2 + suma)

print(4*suma)
